sources/asana_dlt/README.md

```diff
@@ -0,0 +1,75 @@
+# Asana
+
+This verified source enables data loading from the Asana API to your preferred destination. It supports loading data from various endpoints, providing flexibility in the data you can retrieve. The following endpoints are available for data loading with this verified source.
+
+Resources that can be loaded using this verified source are:
```

Comment:
there is duplicated sentences. keep the first one
`The following endpoints are available for data loading with this verified source:`